src/core/BufferAttribute.js

```diff
@@ -24,6 +24,7 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 	this.dynamic = false;
 	this.updateRange = { offset: 0, count: - 1 };
+	this.resized = false;
```

Comment:
I think `.resized` should be private... somehow...

Reply:
Maybe `._resized`?

Reply:
is there a convention to indicate private properties? 
_resized looks reasonable.

Reply:
Yes, prefixing with underscore has been a convention used in this library.

Reply:
ok.will do.